db/migrate_cards/data/cards/export

```diff
@@ -1,7 +1,3 @@
 [[export]]
-[[filter_search_metric]]
-[[filter_search_metric+*right+*structure]]
-[[metric+*self+*structure]]
-[[filter_search_topic]]
-[[filter_search_topic+*right+*structure]]
-[[topic+*self+*structure]]
+[[homepage topic metric drag item]]
+[[Company+*type+*structure]]
```

Comment:
Vignesh added a company name next to the logo based on the observation in user testing.